test/journeys/specs/space/basic.js

```diff
@@ -85,41 +97,73 @@ describe(`Widget Space`, () => {
-    const controlsContainer = `.ciscospark-controls-container`;
-    const messageWidget = `.ciscospark-message-wrapper`;
-    it(`has a menu button`, () => {
-      assert.isTrue(browserLocal.isVisible(menuButton));
-    });
 
-    it(`displays the menu when clicking the menu button`, () => {
-      browserLocal.click(menuButton);
-      browserLocal.waitForVisible(activityMenu);
+  describe(`When conversation is established`, () => {
+    before(`wait for conversation to be ready`, () => {
+      const textInputField = `[placeholder="Send a message to ${conversation.displayName}"]`;
+      browserLocal.waitForVisible(textInputField);
     });
 
-    it(`has an exit menu button`, () => {
-      assert.isTrue(browserLocal.isVisible(activityMenu));
-      browserLocal.waitForVisible(exitButton);
-    });
+    describe(`Activity Menu`, () => {
+      it(`has a menu button`, () => {
+        assert.isTrue(browserLocal.isVisible(menuButton));
+      });
 
-    it(`closes the menu with the exit button`, () => {
-      browserLocal.click(exitButton);
-      browserLocal.waitForVisible(activityMenu, 1500, true);
-    });
+      it(`displays the menu when clicking the menu button`, () => {
+        browserLocal.click(menuButton);
+        browserLocal.waitForVisible(activityMenu);
+      });
+
+      it(`has an exit menu button`, () => {
+        assert.isTrue(browserLocal.isVisible(activityMenu));
+        browserLocal.waitForVisible(exitButton);
+      });
+
+      it(`has a message button`, () => {
+        browserLocal.element(controlsContainer).element(messageButton).waitForVisible();
+      });
+
+      it(`has a people button`, () => {
+        browserLocal.element(controlsContainer).element(rosterElements.peopleButton).waitForVisible();
+      });
+
+      it(`hides menu and switches to message widget`, () => {
+        browserLocal.element(controlsContainer).element(messageButton).click();
+        browserLocal.waitForVisible(activityMenu, 1500, true);
+        assert.isTrue(browserLocal.isVisible(messageWidget));
+      });
 
-    it(`has a message button`, () => {
-      browserLocal.click(menuButton);
-      browserLocal.element(controlsContainer).element(messageButton).waitForVisible();
+      it(`closes the menu with the exit button`, () => {
+        browserLocal.click(menuButton);
+        browserLocal.waitForVisible(activityMenu);
+        browserLocal.click(exitButton);
+        browserLocal.waitForVisible(activityMenu, 1500, true);
+      });
     });
 
-    it(`hides menu and switches to message widget`, () => {
-      browserLocal.element(controlsContainer).element(messageButton).click();
-      browserLocal.waitForVisible(activityMenu, 1500, true);
-      assert.isTrue(browserLocal.isVisible(messageWidget));
+    describe(`roster tests`, () => {
+      before(`open roster widget`, () => {
+        browserLocal.click(menuButton);
+        browserLocal.element(controlsContainer).element(rosterElements.peopleButton).click();
```

Comment:
not a huge deal, but you could add this to the menu.js helper file as a function since i've seen it in a few places